javascripting/array-filtering.js

```diff
@@ -0,0 +1,6 @@
+var numbers = [1,2,3,4,5,6,7,8,9,10];
+var filtered = numbers.filter(function evenNumbers(number){
```

Comment:
Does it have something to do with possibly editing the array within the for
loop without affecting the limit of the for loop?
On Apr 11, 2015 7:44 PM, "Jaron Murphy" <jmurphy@updox.com> wrote:

> Oh ok nice!
> On Apr 11, 2015 7:42 PM, "jaybobo" <notifications@github.com> wrote:
>
>> In javascripting/array-filtering.js
>> <https://github.com/paircolumbus/javascripting101/pull/6#discussion_r28199361>
>> :
>>
>> > @@ -0,0 +1,6 @@
>> > +var numbers = [1,2,3,4,5,6,7,8,9,10];
>> > +var filtered = numbers.filter(function evenNumbers(number){
>>
>> No need to name the callback function here unless specified by the drill.
>>
>> var filtered = numbers.filter( function(number){
>>   return number % 2 === 0;
>> });
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/paircolumbus/javascripting101/pull/6/files#r28199361>
>> .
>>
>